backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/MspContractsController.java

```diff
@@ -75,7 +75,11 @@
 	
 	 /**
 	 * Get MSP Coverage info for a Personal Health Number (PHN) of a group Inquiry
-	 * Maps to the legacy R40.
+	 * Maps to the legacy: R40.
+	 * 
+	 * Also used by R37 as the results required for R37 are a subset of those returned for Contract Inquiry so it can return the same result. This 
+	 * does not break overall security as currently all roles with permissions for R37(Get Group Member's Contract Address) also have permission 
+	 * for R40(Contract Inquiry).
 	 *  
 	 * @param contractInquireRequest
```

Comment:
Not related to this commit but the name of the param (in javadoc and method) doesn't match it's class. Inquire vs INquiry.